### DISCUSSION TITLE
Blocks or locations manually emitting light

### DISCUSSION THREAD

**Discussion by JSH32** [Minestom/Minestom#1142] [Help]
How does one make a block or location manually emit light? 
In spigot we had [LightAPI](https://www.spigotmc.org/resources/lightapi.4510/) but I'm wondering if there is a way to do this?

**Comment by KrystilizeNevaDies** [comment]
We do not have a proper light api yet, we are currently working on a lighting engine, once that is done we will have a good ide of what a lighting api will look like, and we will probably implement one.
